src/main/java/seedu/address/logic/parser/ViewAvailableCommandParser.java

```diff
@@ -50,6 +52,17 @@ public ViewAvailableCommand parse(String args) throws ParseException {
                 throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
                         ViewAvailableCommand.MESSAGE_USAGE));
             }
+            //check if dd/MM/yyyy or has any alphabet
+            if (dateStr.contains("/") || dateStr.matches(".*[a-zA-Z]+.*")) {
```

Comment:
This could have just been shortened to check if the format follows the appropriate LocalDateTime format instead of checking for every combination

Reply:
Good catch! But did this initially but it did not catch bugs such as 02/01/2023 so had to implement it like this